### ISSUE TITLE
[Feature request]: Neo.emu evolution

### ISSUE THREAD

**Issue by Snkforever78** [Rakashazi/emu-ex-plus-alpha#1708]
### Describe the feature

Neo.emu is a really good emulator. To make it even more usable, you could insert a Macro function and have the possibility of using Mame cheats where Unibios doesn't reach. Thanks for the work done so far.

**Comment by ErickN13** [comment]
Se você abrir alguma ROM no Neo.emu usando uma Unibios e Apertar Start depois Select bem Rápido, você consegue abrir um menu com Database de cheats

**Comment by Snkforever78** [comment]
Yes I already knew that. But the Mame cheats are on another level, since they both run on Mame perhaps it can also be replicated on Neo.emu. Thanks anyway for the reply.
